src/Link/index.tsx

```diff
@@ -143,22 +151,54 @@ export default class Link extends React.PureComponent<LinkProps, LinkState> {
     this.props.onMouseOut(this.props.linkData.source, this.props.linkData.target, evt);
   };
 
+  getLinkColor(): string[] | string | undefined| null  {
+    const { linkData, pathColorFunc } = this.props;
+    let color = pathColorFunc;
```

Comment:
Maybe adding a type here will fix the error below?
`let color: (string | string[]) = pathColorFunc;`